librosa/output.py

```diff
@@ -196,8 +196,10 @@ def write_wav(path, y, sr, norm=False):
     path : str
         path to save the output wav file
 
-    y : np.ndarray [shape=(n,) or (2,n)]
-        audio time series (mono or stereo)
+    y : np.ndarray [shape=(n,) or (2,n), dtype=np.float]
+        audio time series (mono or stereo).
+
```

Comment:
no need for linebreak?

Reply:
I don't think so?  Other docstrings are formatted like this and it works.

Reply:
oh, my bad. I saw a linebreak during my code review, but that was just my browser wrapping line 199. LGTM